packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -0,0 +1,97 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForReplaceElectorate } from '@agoric/inter-protocol/src/proposals/replaceElectorate.js';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }, opts) => {
+  return harden({
+    sourceSpec: '@agoric/inter-protocol/src/proposals/replaceElectorate.js',
+    getManifestCall: [
+      getManifestForReplaceElectorate.name,
+      {
+        ...opts,
+        economicCommitteeRef: publishRef(
+          install(
+            '@agoric/governance/src/committee.js',
+            '../bundles/bundle-committee.js',
+          ),
+        ),
+      },
+    ],
+  });
+};
+
+const configurations = {
+  MAINNET: {
+    committeeName: 'Economic Committee',
+    // TODO: Update the addresses after confirmation
```

Comment:
TODO when? before or after merging this PR? If after, please add an issue and cite it here.

Reply:
#10194 is the issue to cite.

Reply:
I suppose the :+1: means you agree to add the issue number to this comment, but I don't see any such change yet. I suppose I'll stand by for you to hit the re-review button.

Reply:
The UNTIL form looked good. Did it go away?

Reply:
Reinstated. Just me messing stuff up by going back and forth between the committee and charter branches :D